templates/flipcard.jsx

```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
               `component__item flipcard__item item-${index} ${_flipDirection}`,
             ])}
             key={index}
+            tabIndex='0'
           >
             <div
               className='flipcard__item-face flipcard__item-front'
               role='button'
-              tabindex='0'
+              tabIndex='0'
```

Comment:
I dont think the I of index needs to be capitalised, please refer to this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Thats the mistake I made too Eleanor. HTML-wise you are correct, but React needs the camel case version - and then it changes this to the lowercase for output